index.html

```diff
@@ -108,34 +107,34 @@ <h2 style="font-family: 'Poppins', sans-serif;">
         </section>
 
         <!------START SECTION PROBLEM------->
-        <section class="bgF pt100">
+        <section class="bgDB pt100">
             <span id="sec2">
-            <h8 class="labeltag bgLR colorF">PROBLEM</h8>
+            <h8 class="labeltag bgLR colorDB text-wide">PROBLEM</h8>
             </span>
-            <div style="margin: 0 0 0 30px;">
+            <div>
                 <div class="container pt40 pb75">
                     <div class="row pb20">
-                        <div class="col-md-6 col-md-pull-0 col-sm-7">
-                            <h1 class="large mb40 mb-xs-16">O&amp;P has Obstacles</h1>
-                            <h5 class="mb-xs-16">In 2016, we needed a 30% increase in O&amp;P professionals to meet unmet demand. <strong>By 2025, it will be 60%</strong>.</h5>
-                            <h5 class="mb-xs-16">Cost containment keeps prices at a minimum while demand from your clinicians keeps increasing.</h5>
-                            <h5 class="mb-xs-16">How do you keep your O&amp;P business profitable? The only way is to increase <strong>operational efficiency</strong>, but O&amp;P is bogged down by inefficient practices like manual device ordering and fitting.</h5>
+                        <div class="col-md-6 col-sm-7">
+                            <h1 class="large mb40 mb-xs-16 colorF">O&amp;P has Obstacles</h1>
+                            <h5 class="mb-xs-16 colorF">In 2016, we needed a 30% increase in O&amp;P professionals to meet unmet demand. <strong>By 2025, it will be 60%</strong>.</h5>
+                            <h5 class="mb-xs-16 colorF">Cost containment keeps prices at a minimum while demand from your clinicians keeps increasing.</h5>
+                            <h5 class="mb-xs-16 colorF">How do you keep your O&amp;P business profitable? The only way is to increase <strong>operational efficiency</strong>, but O&amp;P is bogged down by inefficient practices like manual device ordering and fitting.</h5>
                         </div>
                         <!------START PINK BOX RIGHT------->
                         <div class="col-md-6 col-sm-5 col-md-push-1 col-sm-pull-1">
                             <div class="container mt30 accentboxP">
                                 <ul>
                                     <li>
-                                        <h4 class="colorLR">excessive inefficiencies</h4>
+                                        <h4 class="colorLR inconsolata">excessive inefficiencies</h4>
```

Comment:
@allytart This is running off the page. 

[https://www.dropbox.com/s/2yum8vq82phut3f/Screenshot%25202017-02-02%252018.07.50.png?dl=0](url)